docfx_project/ziti/quickstarts/network/hosted.md

```diff
@@ -2,35 +2,34 @@
 
 You can absolutely choose to host your [Ziti Network](xref:zitiOverview#overview-of-a-ziti-network) anywhere you like.
 It is not necessary for the server to be on the open internet. If it works better for you to deploy Ziti on your 
-own network, great, do that.  The only requirement to be aware of is that every piece of the a network will need to 
-be able to communicate to the controller at least one edge router. If these components are all on your local network,
-that's perfectly fine. Just remember, the only components able to participate on this network will need to be able to
-contact the controller and at least one edge router.
+own private network, great, do that.  The only requirement to be aware of is that every piece of the a network will need to 
+be able to communicate to the Ziti Edge i.e. the control plane that is composed of the controller's edge-client REST API and 
+at least one edge router.
 
-With all that said, if you have a server on the open internet, or you are going to provision one to stand up Ziti that's
-ideal. With a Zero Trust overlay network provided by Ziti, you can rest assured that your traffic is safe even when 
+It's ideal for self-hosting Ziti to have those TLS-hardened control plane servers available at a routeable address on the
+internet. With a Zero Trust overlay network provided by Ziti, you can rest assured that your traffic is safe even when 
 using commodity internet. You also do not need to worry about being on a network you trust as all networks are 
-considered untrustworthy, even your work/home network!
+considered untrustworthy, even your private work or home network!
 
 ## Installation
 
 When starting out deploying a [Ziti Network](xref:zitiOverview#overview-of-a-ziti-network), we recommend you follow
-and use the `expressInstall` function provided by the OpenZiti project. Once you're familiar with the network and 
+and use the `expressInstall` function provided by the Ziti project. Once you're familiar with the network and 
 the configuration options available you'll be better equipped to make changes. 
 
 ### Firewall
 
-The first issue you will need to deal with is opening some ports. A network will consist of at least one controller and 
-at least one edge router. Both of these components will require ports to be open. For the controller you will need to 
-open two ports through your firewall, one port for the REST API, and one the control plane. Edge routers will also 
-require two ports open. One for the links created between routers to form the mesh network, and one port for incoming 
-client connections.
+The first issue you will need to deal with is opening some ports. The Ziti network's control plane is compose of a

```

Comment:
Ziti network --> network. it's better imo here to not be so specific. it's a fine line between slapping "openziti/ziti" everywhere and not